client/asset/dcr/dcr.go

```diff
@@ -2609,6 +2613,9 @@ func (dcr *ExchangeWallet) SwapConfirmations(ctx context.Context, coinID, contra
 		return 0, false, err
 	}
 
+	ctx, cancel := context.WithTimeout(ctx, confCheckTimeout)
+	defer cancel()
```

Comment:
Now I'm reconsidering timeouts for the dcr and spv wallets if they need to do a cfilters scan because those can be more time consuming.

Actually, did you mean to skip BTC's timeout in this PR?

Reply:
It looked like btc didn't use the context. Does it?

Reply:
https://github.com/decred/dcrdex/blob/11fe415b1218463254f5e9ee2cdce6e2c4480416/client/asset/btc/btc.go#L2766-L2769

Reply:
Ah, good point.  We have a TODO to update `call` so a context gets used, but you're right it's not used now.